features/firstPartyData.md

```diff
@@ -20,37 +20,39 @@ These conventions aren't implemented by all adapters. Please
 check with each of your bidders to make sure they're reading first
 party data from the standard Prebid locations.
 
+{: .alert.alert-warning :}
+This document covers the method of specifying First Party Data as of Prebid.js 4.xx, which
+we recommend all publishers use. In previous versions of Prebid.js, the interface was
+to set `fpd` values. We now use the more generic `ortb2` interface,
+which can be used for more than just First Party Data.
+
 ## How It Works
 
-Here's a summary of how first party data (FPD) works:
+Publishers supply First Party Data (FPD) by specifying attributes as
+configuration or on a Prebid.js AdUnit. Here's a summary of how it works:
 
 ![First Party Data Summary](/assets/images/flowcharts/FirstPartyData-Summary.png){: .pb-lg-img :}
```

Comment:
Not really sure how useful this diagram is. There's no explanation of what's happening with bidders A and B.

Reply:
Ok, diagram removed.